index.js

```diff
@@ -36,10 +69,47 @@ function Person() {
         + The `drive` method should return a string "I ran out of fuel at x miles!" x being `odometer`.
 */
 
-function Car() {
+function Car(model, milesPerGallon) {
+  this.model = model;
+  this.milesPerGallon = milesPerGallon;
+  this.tank = 0;
+  this.odometer = 0;
+}
+
+Car.prototype.fill = function(gallons){
+  return this.tank = this.tank + gallons;
+}
+
+//Below is testing to make sure Car and .fill() works:
+const car1 = new Car('Subaru', 20);
 
+// console.log(car1)
+// console.log(car1.fill(6))
+// console.log(car1)
+
+
+Car.prototype.drive = function(distance){
+  this.odometer = this.odometer + distance;
```

Comment:
In this case if you have gas in the tank enough for 5 miles but you want to drive 10 mins this function will add 10 miles to odometer even though it will only be able to go 5 miles. You might want to take a look at this function to double check that.